{"name": "test_auth_user", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-04-23_11-01-53.log'", "trace": "self = <tests.test_auth.TestUserAuth object at 0x1026dd250>\n\n    def setup(self):\n        data = {\n            'email': 'vinkotov@example.com',\n            'password': '1234'\n        }\n    \n>       response1 = MyRequests.post(\n            \"/user/login\",\n            data=data\n        )\n\ntests/test_auth.py:20: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nlib/my_requests.py:11: in post\n    return MyRequests._send(url,data,headers,cookies,'POST')\nlib/my_requests.py:34: in _send\n    Logger.add_request(url, data, headers, cookies, method)\nlib/logger.py:27: in add_request\n    cls._write_log_to_file(data_to_add)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ncls = <class 'lib.logger.Logger'>\ndata = \"\\n-------\\nTest: tests/test_auth.py::TestUserAuth::test_auth_user (setup)\\nTime: 2023-04-23 11:01:53.659708\\nRequest ...gin\\nRequest data: {'email': 'vinkotov@example.com', 'password': '1234'}\\nRequest headers: {}\\nRequest cookies: {}\\n\\n\"\n\n    @classmethod\n    def _write_log_to_file(cls, data: str):\n>       with open(cls.file_name, 'a', encoding='utf-8') as logger_file:\nE       FileNotFoundError: [Errno 2] No such file or directory: 'logs/log_2023-04-23_11-01-53.log'\n\nlib/logger.py:10: FileNotFoundError"}, "description": "This test successfully authorize user by email and password", "start": 1682236913659, "stop": 1682236913659, "uuid": "3c0c5db2-4dff-4028-8d52-b9b3ad74a0a0", "historyId": "dadb9f82538e912234592dc4e33acc8c", "testCaseId": "ee0d3c284ef17e3bfe9f7dd18d65d110", "fullName": "tests.test_auth.TestUserAuth#test_auth_user", "labels": [{"name": "epic", "value": "Authorization cases"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "subSuite", "value": "TestUserAuth"}, {"name": "host", "value": "MacBook-Pro-Renat.local"}, {"name": "thread", "value": "9839-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}